compiler/rustc_hir/src/def.rs

```diff
@@ -31,6 +32,7 @@ pub enum CtorKind {
     Fictive,
 }
 
+/// An attribute that is not a macro; e.g., `#[inline]` or `#[rustfmt::skip]`.
```

Comment:
Is it really helpful to add tautological comments like this just for the sake of it?
The name tells what it is, and comments on variants already describe the individual cases.

Reply:
I did this for two reasons:

1. We use a lot of abbreviations in rustc that can be confusing to newcomers.
2. I wanted to include an example of this kind of attribute, but having the entirety of the docs be the word `#[inline]` is pretty confusing.

Reply:
I guess it's a question of who these comments are optimized for, this *may* be useful for a person looking at this enum for the first time, but for anyone else using this as a reference/reminder (which arguably happens more often) tautologies and walls of text are distractions and a nuisance.